src/Http/Resources/DepartmentResource.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace dnj\Ticket\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class DepartmentResource extends JsonResource
```

Comment:
Why do you customize this JsonResource?
There is anything wrong with original JsonResource behavior?

Reply:
There was no problem.
I have now reverted to the original JsonResource